### ISSUE TITLE
User suggestion: add kinetic and gravitational energy graphs

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/gravity-and-orbits#270]
From a user on twitter:

> @PhETsims hi PhET, please please can you add kinetic and gravitational energy graphics to your Gravity and Orbits simulations. Lots of love, Australian Physics teachers :)

**Comment by samreid** [comment]
@arouinfar or @ariel-phet do you want to work on this before the upcoming publication?

**Comment by arouinfar** [comment]
@samreid I think this would take quite a bit of design and development resources, which I don't think we have any bandwidth for right now. My inclination is to defer, but I'll let @ariel-phet make that call.

**Comment by ariel-phet** [comment]
Yes, I think it would be wise to continue to defer this issue, although it is a good suggestion, it would likely be a whole new screen.
